README.md

```diff
@@ -1,267 +0,0 @@
-# Horondi
```

Comment:
Why have you removed the `README` file, please, revert this changes

Reply:
returned the file README, it was mistake